### DISCUSSION TITLE
Jupyter notebook code (in cell) wrongly displayed as a long string instead of separate lines

### DISCUSSION THREAD

**Discussion by kanru-wang** [community/community#107495] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

On Github website, code in some of my Jupyter notebooks recently began to display weirdly.

Specifically, in a code cell, code lines are displayed as **a long string** instead of **seperate lines**.

For example, https://github.com/kanru-wang/Graph_Neural_Network/blob/main/Node_Classification_Simple.ipynb

and also https://github.com/kanru-wang/Udemy_GAN_and_Diffusion_models/blob/main/CelebA_WGAN.ipynb

I remember a few months ago these Jupyter notebooks were displayed properly, and I haven't made changes since then.

I have tried a few web browsers and saw the same issue.

I have checked the underlying JSON files of the affected Jupyter notebooks, but the newline characters "\n" are in place, and the formatting seems correct.

Thank you.

**Comment by kanru-wang** [comment]
I asked the same question on Stackoverflow, [and got some helpful answer ](https://stackoverflow.com/questions/77982399).

Please let me know if you have another solution. Thank you.

**Comment by zidnyyasrah** [comment, marked as answer]
There have been occasional reports of GitHub rendering issues that can cause Jupyter notebook code to display incorrectly. This is usually a temporary issue that is resolved by GitHub within a short period.
